cv.tex

```diff
@@ -44,92 +50,85 @@
+\location{Major GPA: 3.44 / 4.0}
 
 \sectionspace % Some whitespace after the section
 
-\descript{BS in Computer Science}
-\location{Expected May 2014 | Ithaca, NY}
-Conc. in Software Engineering \\
-College of Engineering \\
-Dean's List (All Semesters) \\
-\location{ Cum. GPA: 3.92 / 4.0 \\
-Major GPA: 3.94 / 4.0}
-
-\sectionspace % Some whitespace after the section
 
+%------------------------------------------------
+% Certifications
 %------------------------------------------------
 
-\subsection{La Martiniere for Boys}
-
-\location{Grad. May 2011 | Kolkata, India}
-
+\section{Certifications}
+\subsection{\href{https://www.giac.org/}{GIAC} (Expired)}
+2017 \href{https://www.giac.org/certifications/cyber-threat-intelligence-gcti/}{GCTI} (\#1144) \\
+2017 \href{https://www.giac.org/certifications/certified-forensic-analyst-gcfa/}{GCFA} (\#13368) \\
+2019 \href{https://www.giac.org/certifications/web-application-penetration-tester-gwapt/}{GWAPT} (\#6137) \\
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
-% Links
+% Skills
 %------------------------------------------------
 
-\section{Links} 
+\section{Skills}
 
-Github:// \href{https://github.com/deedydas}{\bf deedydas} \\
-LinkedIn:// \href{https://www.linkedin.com/in/debarghyadas}{\bf debarghyadas} \\
-YouTube:// \href{https://www.youtube.com/user/DeedyDash007}{\bf DeedyDash007} \\
-Twitter:// \href{https://twitter.com/debarghya_das}{\bf @debarghya\_das} \\
-Quora:// \href{https://www.quora.com/Debarghya-Das}{\bf Debarghya-Das}
+\subsection{Programming}
+\href{https://aws.amazon.com/}{AWS} \textbullet{} Python \textbullet{} \textbf{\href{https://docs.panther.com/}{Panther}} \textbullet{} SQL \\
+\location{Working Proficiency:}
+C \textbullet{} \href{https://console.cloud.google.com/}{GCP} \textbullet{} Java \textbullet{} Javascript \textbullet{} PHP \textbullet{} \textbf{\href{https://www.oracle.com/cloud/}{OCI}} \\
+\location{Familiar:}
+Boto3 \textbullet{} JSON \textbullet{} \LaTeX\ \textbullet{} Pandas \\
+PySpark \textbullet{} Selenium \textbullet{} YAML \\
+
+\subsection{Analytical}
+AWS Security \textbullet{} Databricks \textbullet {} Snowflake \\
+Suricata \textbullet{} Splunk\\
+\location{Forensic Proficiency:}
+.eml/.msg \textbullet{} Linux \textbullet{} MacOS\textbullet{} Windows \\
+\location{Logs of Familiarity:}
+AWS CloudTrail \textbullet{} GuardDuty \textbullet{} CrowdStrike Falcon Data Replicator \\
+Github Audit \textbullet{} Okta System \textbullet{} Zscaler \\
 
 \sectionspace % Some whitespace after the section
 
+
 %------------------------------------------------
-% Coursework
+% Awards
 %------------------------------------------------
 
-\section{Coursework}
-
-\subsection{Graduate}
-
-Advanced Machine Learning \\
-Open Source Software Engineering \\
-Advanced Interactive Graphics \\
-Compilers + Practicum \\
-Cloud Computing
+\section{Awards}
+2023 Dropbox Team TACO "Capitan" \\
```

Comment:
(JC):
The two opening quotes are closing quotes. “ versus ”.

Reply:
fixed. these should just be standard doublequotes but i explicitly set them to open/closed with `option + [`/`]`, seems to work in my preview